### PR TITLE
Fix postinstall steps.

### LINKED ISSUE #9024: Unable to start a demo page
### Environment Information 
 
- __Package version(s)__: current master 
- __Browser and OS versions__: MacOS 10.14.4, node 8.11.3, npm 6.4.0

### Please provide a reproduction of the bug in a codepen:
 
README.md says that to run a demo page I just need to:

1. `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
2. `cd office-ui-fabric-react`
3. `npm install`
4. `npm start`

#### Actual behavior:
 
Probably caused by #8844, `npm start` fails with errors similar to the following one:
```
ERROR in ./src/components/pages/ActivityItemPage.tsx
Module not found: Error: Can't resolve '@uifabric/api-docs/lib/pages/office-ui-fabric-react/ActivityItem.page.json' in '/Users/mistastn/devel/office-ui-fabric-react/apps/fabric-website-resources/src/components/pages'
 @ ./src/components/pages/ActivityItemPage.tsx 7:124-209
 @ ./src/AppDefinition.tsx
 @ ./src/index.tsx
 @ multi /Users/mistastn/devel/office-ui-fabric-react/common/temp/node_modules/.registry.npmjs.org/webpack-dev-server/3.1.14/webpack@4.29.5/node_modules/webpack-dev-server/client?http://localhost:4322 ./src/index.tsx
```

I have to run `npm run build` before `npm start` to run the demo page. 

#### Expected behavior:
 
README.md in sync with what I need to do.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: (Blocking, High, Normal, Low)

### PR DESCRIPTION
**Pull request by jdhuntington**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #9024
- NA Include a change request file using `$ npm run change`

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9028)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,11 +10,11 @@
   "scripts": {
     "_rushInstall": "node common/scripts/install-run-rush.js install --purge",
     "_rushBuild": "node common/scripts/install-run-rush.js build",
-    "_rushBuildToExperiments": "node common/scripts/install-run-rush.js build --to @uifabric/experiments",
+    "_rushBuildToFabricResources": "node common/scripts/install-run-rush.js build --to @uifabric/fabric-website-resources",
```
Comment (ecraig12345) [inline package.json]: Is fabric-website-resources before or after experiments?

Comment (khmakoto) [reply package.json]: I think it is before, I think the correct thing would be to use `fabric-website` here.

Comment (jdhuntington) [reply package.json]: This is before. I don't believe experiments is required in order to run `npm start`, and would prefer not to add unnecessary steps as this is already a long process.

Comment (ecraig12345) [reply package.json]: Someone wanted to build to experiments and pretty recently made the change to do that. You should look at the change history and talk to whoever it was before reverting.

**Pull request merged**